apps/frontend/src/pages/quiz/libs/components/analyzing/analyzing.tsx

```diff
@@ -19,6 +30,17 @@ const Analyzing: React.FC<Properties> = ({ onNext }: Properties) => {
 		secondParagraphDelay,
 	);
 
+	const handleNextStep = useCallback(() => {
+		if (!user) {
+			return;
+		}
+
+		const updatedUser = { ...user };
+		updatedUser.hasAnsweredOnboardingQuestions = true;
```

Comment:
Instead create a new action, which will set hasAnsweredOnboardingQuestions to true